webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyState.scss

```diff
@@ -0,0 +1,7 @@
+.pf-c-empty-state {
+  margin-top: 100px;
+
+  .empty-state-icon {
+    font-size: xxx-large;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/font-size

it seem that the `xxx-large` is not supported in all latest browser, lets use a pixel size,
or even better - patternfly's font size variable

Reply:
you mean pf4 global CSS variable right? https://www.patternfly.org/v4/documentation/overview/global-css-variables

Reply:
exactly, something like `--pf-global--icon--FontSize--xl`
you can import it from the patternfly node module in the .scss file as `@import "~patternfly/sass/app";` to the specific location of that var.

Reply:
gotcha! thanks a lot @laviro :+1: